### ISSUE TITLE
v2.7.0 broke backward compatibility of the Artifactory constructor

### ISSUE THREAD

**Issue by Tpt** [anancarv/python-artifactory#197]
Thank you for your great library!

Sadly, it seems to me commit 210e3aac7910cec68a5f4ebfd3f5d30e6ff7a7a8 shipped in v2.7.0 added a new parameter (`access_token`) in the middle of the `Artifactory` class constructor, breaking backward compatibility.

**Comment by anancarv** [comment]
> Thank you for your great library!
> 
> Sadly, it seems to me commit [210e3aa](https://github.com/anancarv/python-artifactory/commit/210e3aac7910cec68a5f4ebfd3f5d30e6ff7a7a8) shipped in v2.7.0 added a new parameter (`access_token`) in the middle of the `Artifactory` class constructor, breaking backward compatibility.

Hey @Tpt,
Can you please provide more details or some logs. The `access_token` param is Optional and should default to None is not set.

**Comment by Tpt** [comment]
Sure!

In `pyartifactory` 2.6, one could instantiate with an `Artifactory` object with `Artifactory(MY_URL, MY_AUTH, False)` to disable the "verify" option. This call will fail in 2.7 because the 3 argument is now `access_token` and not `verify` anymore, leading to the assignation of `False` to `access_token`.

**Comment by anancarv** [comment]
Ahh I see, I suggest you to use `keyword arguments` instead of` positional arguments`. In other words, you can do:

```python
art = Artifactory(url="ARTIFACTORY_URL", auth=('USERNAME','PASSWORD_OR_API_KEY'), verify=False)
```

**Comment by Tpt** [comment]
Thank you! It's what I already did.

**Comment by anancarv** [comment]
The issue is fixed in the new version `v2.7.1`
